docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,274 @@
+# Deploying a Mesos Cluster with Calico on CentOS7
+
+In these instructions, we will install Mesos and Calico Networking on baremetal CentOS/Red Hat Enterprise Linux 7 from rpm packages. These packages can be built from the Mesosphere [net-modules repo](https://github.com/mesosphere/net-modules) and will install the following processes:
+
+ * Mesos Master
+ * Mesos Agent
+ * net-modules
+ * Calico
+
+To simplify deployment, ZooKeeper and etcd will run in Docker containers on the Master host.
+
+## Preparation
+
+For the installation of Mesos + Calico, you will need Docker 1.7+. You must also verify your host has a unique fully qualified domain name. Lastly, we recommend disabling the firewall on the Mesos communication ports.
+
+The instruction below will walk you through the steps.
+
+### Prepare External Services
+
+You will need to run one instance of both, etcd and ZooKeeper. Most Mesos cluster will run the backend services in a cluster external to their core Mesos cluster. For simplicity's sake, we will be launching ZooKeeper and etcd on the master host.
```

Comment:
Zookeeper and etcd serve as the backend datastores for Mesos and Calico, respectively. Most Mesos clusters will run these services in a seperate cluster, seperate from the Masters and Agents which read from them. 

As a simple proof-of-concept, we'll cover running one instance of each as Docker containers on our Mesos-Master. However, you are welcome to run them wherever you choose.